src/main/java/Event.java

```diff
@@ -0,0 +1,34 @@
+public class Event extends Task{
+    protected String TYPE = "event";
+    protected String startTime;
+    protected String endTime;
+
+
```

Comment:
Consider adding @Override for clarity that this is an overridden function from its superclass.